uk.co.compucorp.civicrm.hrleaveandabsences/scss/overrides/style/_buttons.scss

```diff
@@ -0,0 +1,14 @@
+.crm-button .crm-i {
+  top: .6em;
+  left: .6em;
+}
+
+.crm-button-type-next {
+  @include button-outline-variant(white, $brand-primary);
```

Comment:
Please add a line-break before `@include`

Reply:
before @include ? like this:

```scss
.crm-button-type-next {

  @include button-outline-variant(white, $brand-primary);
}
```

Reply:
Yep. No line-break needed if you list styles, but if you include smth or nest, then a line-break is needed. Like here:

![image](https://user-images.githubusercontent.com/3973243/29932093-35e97862-8e6b-11e7-950c-d33785cd6f1f.png)